pywren/wrenhandler.py

```diff
@@ -32,6 +32,8 @@
 
 PROCESS_STDOUT_SLEEP_SECS = 2
 
+TMP_MIN_FREE_SPACE_BYTES = 10000000
```

Comment:
nit - can we add a comment above this "Leave at least 10MB free space" or something like that

Reply:
This could be `1e8` instead?